perun-core/src/main/java/cz/metacentrum/perun/core/bl/MembersManagerBl.java

```diff
@@ -313,8 +312,9 @@
 	 * @throws AlreadySponsoredMemberException if member was already flagged as sponsored
 	 * @throws UserNotInRoleException if sponsor has not right role in the member's VO
 	 * @throws InternalErrorException if something unexpected happened
+	 * @throws sponsoredMember is already sponsored by User and his sponsorship is still active
```

Comment:
Missing ``AlreadySponsorException`` at the start.

Reply:
Should be fixed now.